addons/io/org.openhab.io.homekit/README.md

```diff
@@ -16,7 +16,7 @@ Requirements beyond this are not clear, and Apple enforces limitations on eligib
 At the very least, you cannot use repeating (111-11-111) or sequential (123-45-678) pin codes.
 If your home network is secure, a good starting point is the pin code used in most sample applications: 031-45-154.
 
-Other settings, such as using Fahrenheit temperatures, customizing the thermostat heat/cool/auto modes, and specifying the interface to advertise the Homekit bridge on are also illustrated in the following sample:
+Other settings, such as using Fahrenheit temperatures, customizing the thermostat heat/cool/auto modes, and specifying the interface to advertise the Homekit bridge, which can be edited in PaperUI in standard mode, on are also illustrated in the following sample:
```

Comment:
Consider, "advertise the Homekit bridge (which can be edited in PaperUI standard mode) are also...